src/components/Tasks/TasksToggle.js

```diff
@@ -0,0 +1,29 @@
+import "./TasksToggle.css";
+import React, { useState } from "react";
+
+function TasksToggle(props) {
+  const [FilterStatus, setFilteredStatus] = useState("");
```

Comment:
Same as above! You don't need this state. get the value from its parent component